### ISSUE TITLE
netCDF files cannot be opened with xarray

### ISSUE THREAD

**Issue by madrouin** [ACTRIS-CCRES/raw2l1#9]
example for file `test_cl31_20141030_000002.nc` created by tests:

```python

~/.local/src/anaconda3/lib/python3.6/site-packages/xarray/coding/times.py in decode(self, variable, name)
    398 
    399         enable_cftimeindex = OPTIONS['enable_cftimeindex']
--> 400         if 'units' in attrs and 'since' in attrs['units']:
    401             units = pop_to(attrs, encoding, 'units')
    402             calendar = pop_to(attrs, encoding, 'calendar')

TypeError: argument of type 'numpy.int32' is not iterable
```

**Comment by madrouin** [comment]
bug is caused by `units = 1` (for variables without units in CF conventions) being typed as integer by raw2l1.

**Comment by madrouin** [comment]
solved by 5dcfc2f1a98b9ee6200907ca7de5b6d0661940f8

**Comment by madrouin** [comment]
closed
